TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/LinearSlideSubsystem.java

```diff
@@ -90,9 +109,9 @@ public RunPIDCommand(LinearSlideSubsystem linearSlideSubsystem) {
 
         @Override
         public void execute() {
-            linearSlideSubsystem.telemetry.addData("Target position", linearSlideSubsystem.getTargetPosition());
-            linearSlideSubsystem.telemetry.addData("Current position", linearSlideSubsystem.getCurrentPosition());
-            linearSlideSubsystem.runPIDPosition(linearSlideSubsystem.getTargetPosition());
+            linearSlideSubsystem.telemetry.addData("Slide Target position", linearSlideSubsystem.getTargetPosition());
```

Comment:
I get how it's nice to have the differentiation, but this was really just meant for testing, we may not even keep it.